packages/web-components/fast-components/src/anchor/index.ts

```diff
@@ -50,7 +50,7 @@ export class Anchor extends FoundationAnchor {
      * @internal
      *
      */
-    public defaultSlottedContentChanged(oldValue, newValue): void {
+    public defaultSlottedContentChanged(oldValue: string, newValue: string): void {
```

Comment:
wouldn't it be `HTMLElement[] | undefined`?

Reply:
`HTMLElement[] | undefined` would be better. I suggested `unknown` in this case since the underlying value for `defaultSlottedContent` isn't filtered until it reaches this function, and it's actually `Node[]` when coming from the `slotted` directive. The filter in this function should either be moved to the `slotted` call in the template, or the property `defaultSlottedContent` should be changed to type `Node[]` in the base class.

Reply:
On further investigation, this specific function is handling cases where only a single `SVGElement` is present, so `HTMLElement[]` isn't the right type.